mochify/index.js

```diff
@@ -62,17 +62,32 @@ async function mochify(options = {}) {
   return { exit_code };
 }
 
-function resolveMochifyDriver(name = 'puppeteer') {
+function resolveMochifyDriver(name) {
+  if (!name) {
+    throw new Error(
+      'Specifying a driver option is required. Mochify drivers need to be installed separately from the API or the CLI.'
+    );
+  }
+
+  let driverModule;
   try {
     // eslint-disable-next-line node/global-require
-    return require(`@mochify/driver-${name}`);
+    driverModule = require(`@mochify/driver-${name}`);
   } catch (err) {
     if (err.code !== 'MODULE_NOT_FOUND') {
       throw err;
     }
     // eslint-disable-next-line node/global-require
-    return require(name);
+    driverModule = require(name);
   }
+
+  if (!driverModule || typeof driverModule.mochifyDriver !== 'function') {
```

Comment:
This could theoretically be made even stricter if we checked for the `length` but then again how knows what people will try to pass here. I think it's ok to know it has the proper name and can be called.

Reply:
There's already the word `mochify` in the signature. I agree this is enough.